packages/smart-contracts/test/contracts/StorageFeeCollector.test.ts

```diff
@@ -167,9 +167,7 @@ describe('contract: StorageFeeCollector', () => {
       );
       const contentSize = BigNumber.from(1000);
 
-      await expect(storageFeeCollector.getFeesAmount(contentSize)).to.be.revertedWith(
-        'SafeMath: multiplication overflow',
-      );
+      await expect(storageFeeCollector.getFeesAmount(contentSize)).to.be.reverted;
```

Comment:
From my experience, this is risky, I found a few tests failing for the wrong reason previously. Can you instead replace with the expected error message ?

Reply:
The new safeMath mechanism does not give any message.

Reply:
Maybe since safeMath is a default we can remove these low level tests ?